rasa/core/actions/forms.py

```diff
@@ -223,7 +224,7 @@ def extract_other_slots(
                     # check whether the slot should be
                     # filled from trigger intent mapping
                     should_fill_trigger_slot = (
-                        tracker.active_loop.get("name") != self.name()
+                        tracker.active_loop.get(NAME) != self.name()
```

Comment:
since there is ongoing discussion whether to use object for `active_loop` I'll revert to using string, in order not to do double job creating constants that will be removed

Reply:
renamed all constants for active loop to `LOOP_...` e.g. `NAME` to `LOOP_NAME`